src/utils/config/kubernetes.js

```diff
@@ -6,26 +6,50 @@ import { KubeConfig } from "@kubernetes/client-node";
 
 import checkAndCopyConfig, { CONF_DIR, substituteEnvironmentVars } from "utils/config/config";
 
-export default function getKubeConfig() {
+const extractKubeData = (config) => {
+  // kubeconfig
+  const kc = new KubeConfig();
+  kc.loadFromCluster();
+
+  // route
+  let route = "ingress";
+  if (config?.route === "gateway") {
+    route = "gateway";
+  }
+
+  // traefik
+  let traefik = true;
+  if (config?.traefik === "disable") {
+    traefik = false;
+  }
```

Comment:
```suggestion
  const route = config?.route ?? "ingress";
  const traefik = config?.traefik !== "disable";
```

or even `const route = config?.route === "gateway" ? "gateway" : "ingress";` if you want to be explicit